src/server/scripts/Outland/HellfireCitadel/HellfireRamparts/boss_vazruden_the_herald.cpp

```diff
@@ -193,7 +193,7 @@ class boss_nazan : public CreatureScript
                 uint32 Fly_Timer;
                 uint32 Turn_Timer;
                 bool flight;
-                ObjectGuid VazrudenGUID;
+                Creature* Vazruden;
```

Comment:
never store a Creature* pointer. if a gm despawns it with a command, it will crash the server

Reply:
This one i don't have the knowledge to fix. Maybe someone can find a better way to do that.

Reply:
well the way to fix it is to use the old code which you changed from ObjectGuid to Creature* . But if you don't know how to fix it, maybe you don't know what the changes in this PR do either ?
Maybe it's better if the original author of the changes opens the PR so he can implement the required changes.

Reply:
I am guessing that this code was contributed to Aokromes' GitHub or GitLab repo, but considering the implication, that user might not want to contribute to the TC repository (as I said, I am just guessing, but it is based on previous experience from the TC issue tickets).